modules/graphql/server-ts/index.ts

```diff
@@ -0,0 +1,29 @@
+import schemaDocument from './schema.graphql';
+import createResolvers from './resolvers';
+import { isApiExternal } from '@gqlapp/core-common';
+import ServerModule, { MiddlewareFunc } from '@gqlapp/module-server-ts';
+
+import graphiqlMiddleware from './graphiql';
+import createApolloServer from './createApolloServer';
+import { MakeGQLContextCreator } from './types';
+
+const middleware: MiddlewareFunc = (app, appContext, { createGraphQLContext, schema }) => {
+  app.get('/graphiql', graphiqlMiddleware);
+
+  if (!isApiExternal) {
+    const graphqlServer = createApolloServer({ createGraphQLContext, schema });
+    graphqlServer.applyMiddleware({ app, path: __API_URL__, cors: { credentials: true, origin: true } });
+  }
+};
+
+const makeGQLContextCreator: MakeGQLContextCreator = modules => (req, res) => modules.createContext(req, res);
```

Comment:
`createContext` function shouldn't belong to the `ServerModule` anymore. Instead, the new `GraphQLModule` should be created that provides this function.

Reply:
The same is for `schema` and `resolvers` params of the `ServerModule`

Reply:
@evgenijolshanskij Currently `GraphQLModule` is a client-side GraphQL feature modules interface

Reply:
@vlasenko updated the module name.

Reply:
@evgenijolshanskij Yes, sounds good to me, also we can remove client-side `GraphQLModule` to `GraphQLClientModule`, we will not break backwards compatibility by doing this

Reply:
Done.